runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -212,6 +213,13 @@ public void onAnimationChanged(AnimationChanged event)
 				resetTimers();
 				lastAnimation = animation;
 				break;
+			case MAGIC_LUNAR_SHARED:
+				if (graphic != GraphicID.BAKE_PIE) // Don't send an alert unless the graphic matches Bake Pie from the shared animation
```

Comment:
invert this condition as you actually want to start idle timer if BAKE_PIE is graphic id. also you can remove the comment because it is pretty self explanatory